common/app/services/S3.scala

```diff
@@ -163,10 +163,7 @@ object S3FrontsApi extends S3 {
   def listCollectionIds: List[String] = getCollectionIds(s"$location/collection/")
   def putCollectionJson(id: String, json: String) = {
     val putLocation: String = s"$location/collection/$id/collection.json"
-    if (Switches.FaciaToolPutPrivate.isSwitchedOn) {
```

Comment:
Yes, I never switched it on because when I did one time it broke A LOT of things.
I thought you'd at least keep the switch around; you're going to end up doing this regardless of where it gets saved.